vmm/src/config.rs

```diff
@@ -205,6 +207,10 @@ impl fmt::Display for ValidationError {
                 "Number of queues to virtio_net does not match the number of input FDs"
             ),
             VnetReservedFd => write!(f, "Reserved fd number (<= 2)"),
+            NoHarewareChecksumOffload => write!(
```

Comment:
There is also a typo in the name of the error enum value.

Reply:
Thanks, fixed.